service/controller/v1/resource/chartconfig/desired.go

```diff
@@ -3,7 +3,8 @@ package chartconfig
 import (
 	"context"
 
-	"github.com/giantswarm/apiextensions/pkg/apis/core/v1alpha1"
+	corev1 "github.com/giantswarm/apiextensions/pkg/apis/core/v1alpha1"
+	releasev1 "github.com/giantswarm/apiextensions/pkg/apis/release/v1alpha1"
```

Comment:
The alias does not express the truth. When we want to prefix, it should be `releasev1alpha1`. The important path is not `.../release/v1`.

Reply:
Good point.

Renamed all:
applicationv1 > applicationv1alpha1
corev1 > corev1alpha1
releasev1 > releasev1alpha1